recipes/openms-meta/meta.yaml

```diff
@@ -143,14 +145,14 @@ outputs:
      run:
        - {{ pin_subpackage('openms', exact=True) }}
        - comet-ms ==2023010
-       - fido ==1.0
        - luciphor2 ==2020_04_03
-       - msgf_plus ==2021.03.22
+       - msgf_plus ==2023.01.1202
        - pepnovo ==20101117
        - percolator ==3.5
-       - sirius-csifingerid =4.9
-       - thermorawfileparser ==1.3.4
+       - sirius-ms ==5.8.2 # from conda-forge
```

Comment:
I remember that there was an explicit way to refer to a package from a specific channel.

Reply:
```suggestion
       - conda-forge::sirius-ms ==5.8.2
```

Reply:
ah good to know. I am fine with that if it works. Or should I change something?

Reply:
There is only one from conda-forge? Why is this needed?
Also please keep in mind that we prioritise conda-forge over bioconda if a package is in both channels.